personality/ui_gsissh/config.pan

```diff
@@ -0,0 +1,123 @@
+unique template personality/ui_gsissh/config;
+
+#base configuration
+
+"/software/components/chkconfig/service/gsisshd/on" = "";
+"/software/components/chkconfig/service/gsisshd/startstop" = true;
+"/software/components/gsissh/server/port" = GSISSH_PORT;
+
+# Change the GLOBUS SXXsshd file, that is read by ncm-gsissh, in order to 
+#   - use gsisshd name
+#   - SOURCE the environment before starting the server
+
+variable CONTENTS = <<EOF;
```

Comment:
Can you put this in a seperate file and use `file_contents()` please?
That way we can have syntax highlighting and lint-checking on the shell script.